docs/docs/index.md

```diff
@@ -155,16 +155,44 @@ run/debug configuration.
 
 ### VSCode
 
-Add following in the `settings.json`:
+Add the following to `.vscode/settings.json`:
 
 ```json
 {
-  ...
+  "python.pythonPath": "/usr/bin/python",
   "python.autoComplete.extraPaths": ["__pypackages__/<major.minor>/lib"],
   "python.analysis.extraPaths": ["__pypackages__/<major.minor>/lib"]
 }
 ```
 
+Change the Python interpreter path to the one you want to use for the current project.
+
+To allow debugging your project in VSCode, set the `PYTHONPATH` environment variable in your launch configuration, in `.vscode/launch.json`.
+For example, to debug your `pytest` run:
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "pytest",
+            "type": "python",
+            "request": "launch",
+            "module": "pytest",
+            "args": ["tests"],
+            "justMyCode": false,
+            "env": {"PYTHONPATH": "__pypackages__/3.9/lib"}
```

Comment:
If you enabled PEP 582 packages loading, no need to tweak PYTHONPATH

Reply:
I guess you mean this? https://pdm.fming.dev/#enable-pep-582-globally

The problem is that your IDE does not obviously inherit from your shell environments. If I launch VSCode from my shell with `code .` it will work, but if I run it with `Alt+F2+code`, VSCode won't know about the PEP582 sitecustomize.py.

Reply:
OK indeed VSCode launches the process in a shell, so PEP582 is enabled automatically as long as it runs using the same user.

Reply:
I've opted for a collapsible note to document this.

Reply:
Collapsible admonitions require the `pymdownx.details` Markdowm extension, so I replaced it with a classic note. Let me know if you'd prefer the collapsible version.